tests/Feature/ProductFeedTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace BazaarVoice;
+
+use BazaarVoice\Elements\ProductElement;
+use BazaarVoice\Product\Feed;
+use PHPUnit\Framework\TestCase;
+
+class ProductFeedTest extends TestCase
+{
+    /** @test */
+    public function it_generates_an_interaction_feed()
+    {
+        // Set
+        $feed = new Feed();
+        $element = $feed->newFeed('ProductFeed');
+        $product = new ProductElement(12345678, 'Product Name', 'ProductCategoryId123', 'http://localhost/12345678', 'http://localhost/12345678/image');
+        $product2 = new ProductElement(12345679, 'Product Name 2', 'ProductCategoryId123', 'http://localhost/12345679', 'http://localhost/12345679/image');
+        $element->addProduct($product);
+        $element->addProduct($product2);
+        $expectedFeed = file_get_contents('tests/fixtures/product-feed.xml');
+
+        // Actions
+        $result = $feed->printFeed($element);
+
+        // Work aroud nice
+        $result = $this->prepareResultForTesting($result);
+        $expectedFeed = $this->prepareResultForTesting($expectedFeed);
+
+        // Assertions
+        $this->assertEquals($expectedFeed, $result);
+    }
+
+    public function prepareResultForTesting(string $result): string
```

Comment:
I don't find very intuitive manipulate results before asserting them. sounds confusing and can lead to false positives

Reply:
I don't like it too, but I think that in this case, as you see, its a little modification to a better code coverage without the need to inject a dependency ( like Carbon (ohh... Carbon)) only for testing purpose.

Reply:
@brnbp What do you suggest?

Reply:
I also don't find that very amusing

Reply:
I would rather inject the Carbon dependency, nesting str functions confuses a lot